packages/css/gulpfile.js

```diff
@@ -101,43 +107,39 @@ async function createSvgSpritesheet() {
             const id = symbol.attr('id');
 
             const stylesheet = stylesheets.find(s => s.id === id);
-            const iconIds = [
-              id,
-              ...(stylesheet ? stylesheet.variants : []).map(v => `${id}-${v}`)
+
+            const variants = [
+              { selector: id, color: stylesheet ? stylesheet.variants.find(v => v.selector === 'default')?.color : 'currentColor' },
+              ...(stylesheet ? stylesheet.variants.filter(v => v.selector !== 'default') : []).map(v => ({ selector: `${id}-${v.selector}`, color: v.color }))
```

Comment:
`currentColor` is SVG taal, terwijl we hier nog pre-templating zijn. Zoals je `selector` opbouwt is randje, maar `currentColor` is tever.

Wat dacht je van:

```suggestion
              { selector: id, color: stylesheet?.variants.find(v => v.selector === 'default')?.color },
              ...(stylesheet?.variants.filter(v => v.selector !== 'default').map(v => ({ selector: `${id}-${v.selector}`, color: v.color })) ?? [])
```

Dan kan `color` ook type `undefined` zijn. Regel 127 kan je dan voorwaardelijk op `if (variant.color)` doen.